proposals/PlayServicesVersion_Proposal.md

```diff
@@ -0,0 +1,59 @@
+# Play Services Version Proposal
+
+- Status: Proposed
+
+## Overview
+This is a proposal to offer an alternative solution for [CB-13145](https://issues.apache.org/jira/browse/CB-13145).
+
+## Problem
+Plugins using different versions of Google Play Services or the `+` to indicate range cause gradle to error out with the following error:
+
+```
+> Failed to apply plugin [id 'com.google.gms.google-services']
+   > For input string: "+"
+```
+
+## Potential Solution
```

Comment:
Sorry to clarify this is one potential solution. The one from the issue. But the new proposal is actually below.